### PR TITLE
Generic name to object

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
```SUMMARY: none```

#### Purpose of change
Contributes to #36110 and #36611

#### Describe the solution
Change name to object

#### Describe alternatives you've considered
Not changing

#### Testing
Could not spawn:
broken m2 autonomous crows II
broken m490 autonomous crows II
broken police bot
broken riot control bot
broken secubot
broken turret

image
image
image

### CODE REVIEW

File: data/json/items/generic.json [hunk 1]
```diff
@@ -2855,7 +2855,7 @@
     "id": "broken_turret_bmg",
     "symbol": ",",
     "color": "green",
-    "name": "broken M2 autonomous CROWS II",
+    "name": { "str": "broken M2 autonomous CROWS II" },
```
Comment (ZhilkinSerg) [inline data/json/items/generic.json]: I am not sure about that IIs.

File: data/json/items/generic.json [hunk 2]
```diff
@@ -2837,7 +2837,7 @@
     "id": "broken_turret_rifle",
     "symbol": ",",
     "color": "green",
-    "name": "broken M249 autonomous CROWS II",
+    "name": { "str": "broken M249 autonomous CROWS II" },
```
Comment (I-am-Erk) [inline data/json/items/generic.json]:
```suggestion
    "name": { "str": "broken M249 autonomous CROWS II", "str_pl": "broken M249 autonomous CROWS II turrets" },
```

File: data/json/items/generic.json [hunk 3]
```diff
@@ -2846,7 +2846,7 @@
     "id": "broken_crows_m240",
     "symbol": ",",
     "color": "green",
-    "name": "broken M240 autonomous CROWS II",
+    "name": { "str": "broken M240 autonomous CROWS II" },
```
Comment (I-am-Erk) [inline data/json/items/generic.json]:
```suggestion
    "name": { "str": "broken M240 autonomous CROWS II", "str_pl": "broken M240 autonomous CROWS II turrets" },
```

File: data/json/items/generic.json (hunk 1 above)
Comment (I-am-Erk) [inline data/json/items/generic.json]:
```suggestion
    "name": { "str": "broken M2 autonomous CROWS II", "str_pl": "broken M2 autonomous CROWS II turrets" },
```

**Comment by snipercup** [comment]
I think that's all of them. thanks guys

**Pull request merged**